lib/verify-stream.js

```diff
@@ -11,10 +11,10 @@ function isObject(thing) {
   return Object.prototype.toString.call(thing) === '[object Object]';
 }
 
-function safeJsonParse(thing) {
+function safeJsonParse(thing, encoding) {
   if (isObject(thing))
     return thing;
-  try { return JSON.parse(thing); }
```

Comment:
this is not supported by the `JSON.parse` API, see comment here: https://github.com/auth0/node-jws/pull/86#discussion_r1090150451

Ref: `reviver` in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse

Reply:
IMO only the `if (!payload) { return null; }` check should be added by this CR, encoding support is addressed in newer CR #86